fastlane/Fastfile

```diff
@@ -278,7 +278,6 @@ platform :ios do
     version_number = current_version_number
     # making it its own lane until https://github.com/CocoaPods/CocoaPods/issues/11621 is fixed.
     # push_pods 
-    carthage_archive
```

Comment:
I've also removed this from `release`. It's useful to have it in `release_checks`, to make sure that our framework can be built with Carthage`.
However, what we actually release is using `export_xcframework` which builds the frameworks. This will make `release` MUCH faster since we no longer build everything twice as many times (and for each platform / architecture combination).